packages/yoastseo/src/languageProcessing/languages/ja/helpers/getWords.js

```diff
@@ -15,7 +14,7 @@ const segmenter = new TinySegmenter();
  */
 export default function( text ) {
 	// Strips HTML tags and exclude Table of Contents from the analysis.
-	text = stripTags( excludeTableOfContentsTag( text ) );
+	text = sanitizeString( text );
```

Comment:
We can consider doing the same replacement in the default `getWords` functionality.

Moreover, I see that in `scoring/assessments/readability/TextPresenceAssessment.js` we similarly strip the HTML tags and the table of contents, so that one could be replaced as well.

Reply:
I am not sure if this is still in the scope of this issue. This issue focuses on Japanese. But I'll create technical debt issue to improve our code in this regard :)

Reply:
@mhkuu This is [the technical debt issue](https://yoast.atlassian.net/browse/LINGO-1186) :)

Reply:
Cool, thanks!